src/edgeworkers/ew-cli-main.ts

```diff
@@ -113,9 +124,9 @@ program
 
 program
   .command('register <group-identifier> <edgeworker-name>')
-  .description('Register a new EdgeWorker id to reference in Property Manager behavior.')
+  .description('Register a new EdgeWorker Id to reference in Property Manager behavior')
```

Comment:
If you're making a change to these for consistency, can you instead go with "ID"